src/Fhir.Anonymizer.R4.Tool/Fhir.Anonymizer.R4.Tool.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Fhir.Anonymizer.Core\Fhir.Anonymizer.Core.csproj" />
+    <ProjectReference Include="..\Fhir.Anonymizer.R4.Core\Fhir.Anonymizer.R4.Core.csproj" />
```

Comment:
It seems the projects of ADF and command line tools are moved into a file called "tool". Not sure if we keep them in the same folder or 2 separate folders (tools, adf) respectively. If we keep them in the same folder it might be better to rename the projects from tools to command line tools.